tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, **kwargs):
                          default=False,
                          help='restart target instance between tests')
 
+        group.add_option('--locale',
+                         default="undefined",
```

Comment:
I wonder if we could simplify the code, if we put `default='en-US'`, here. We wouldn't need to do checks like 
``` python 
if self.locale != 'undefined'
```
Also, what happens if a user doesn't put that value into the testvars.json? The help message doesn't make it clear. Something like this, looks a bit clearer, I think:
```python
       group.add_option('--locale',
                        default = "en-US",
                        help = "locale for the device, "
                               "This value can be overridden with testvars.json")
```